test/msnodesqlv8/msnodesqlv8.js

```diff
@@ -178,7 +178,7 @@ describe('msnodesqlv8', function () {
 
   describe('msnodesqlv8 connection errors', function () {
     it('login failed', done => TESTS['login failed'](done, /Login failed for user '(.*)'\./))
-    it.skip('timeout (not supported by msnodesqlv8)', done => TESTS.timeout(done, /1000ms/))
+    it.skip('timeout (not supported by msnodesqlv8)', done => TESTS.timeout.call(this, done, /1000ms/))
```

Comment:
I think this will need to be wrapped in a non-arrow anon function as with the tedious.js file

Reply:
Pretty sure this still needs to be `it('timeout', function (done) { TESTS.timeout.call(this, done, /Failed to connect to 10.0.0.1:1433 in 1000ms/) })`